### PR TITLE
GG-34208 AssertionError on getting cache size from the mbean on the inactive cluster

### PR DESCRIPTION
**Pull request by odiszapc**
Port of  https://issues.apache.org/jira/browse/IGNITE-13227

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java [hunk 1]
```diff
@@ -917,7 +917,7 @@ public void onRead(boolean isHit) {
 
     /**
      * Set callback for tx key collisions detection.
-     *
+     *getEntriesStat
```
Comment (sk0x50) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java:920]: Why this change is needed? :)

Comment (odiszapc) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java]: removed
